### ISSUE TITLE
"replaceText" not working 

### ISSUE THREAD

**Issue by SachinAspenify** [singerla/pptx-automizer#82]
Environment - node.js version v21.0.0
OS - Mac 

The package is very useful. Thanks for open-sourcing it. 

I am trying to use the software to generate PPTX file from a template file. The "ReplaceText" function is not working as expected.

I tried to debug the code. But could not find any method that gives me text in an element.

Please find attached sample code I am using for my testing. 
[agreement-ppt.zip](https://github.com/singerla/pptx-automizer/files/13701779/agreement-ppt.zip)

**Comment by singerla** [comment]
I see there is a difference: `err="1"` Could this be related to spell checking? I can remember some really weird situations in the past and finding a solution will make a happy man out of me :laughing:

**Comment by SachinAspenify** [comment]
I did try the solution you mentioned earlier. Somehow I did not work.

What I did now, is that I deleted the placehoder and re-added it to PPT. Now it worked fine. We can close the issue. Thanks for your help.

Is there a way, we view the ppt xml so that it is easy to debug.

**Comment by singerla** [comment]
You could try `slide.modifyElement('testElementName', [XmlHelper.dump]);` 

I'm happy to help :smiley:

**Comment by SachinAspenify** [comment]
I identified the underlying issue. I utilized an existing PowerPoint (PPT) file as a template and included curly brackets around a placeholder string that was already present in the presentation. This unintentionally resulted in the creation of three distinct components ('{', 'placeholder string', '}'), as shown by you above. Consequently, the replaceText function did not function as expected.

It might be beneficial to include this information in a troubleshooting guide to help others avoid encountering a similar issue in the future.
